pmoves-cipher-mcp/cipher_mcp/tools.py

```
@@ -45,6 +52,10 @@ async def pmoves_cipher_store(
             metadata=metadata or {},
         )
 
+        asyncio.ensure_future(
+            emit_memory_stored(memory.id, memory.category, memory.tags or [])
+        )
```

Ruff (0.15.5)

[warning] Store a reference to the return value of `asyncio.ensure_future`

(RUF006)